README.md

```diff
@@ -150,3 +162,34 @@ class WriteComment extends React.Component {
 
 export default connect(mapStateToProps, mapDispatchToProps)(WriteComment);
 ```
+
+## Cache Controlling
+```js
+export default {
+    getArticle: {
+        url: 'http://www.mocky.io/v2/5a0c203e320000772de9664c?:articleId/:userId',
+        method: 'GET',
+        cacheDuration: 60000,
+    }
```

Comment:
Graag nog 2 fake endpoints toevoegen: een voor cache 0 en een voor geen cache. Graag achter elke cacheDuration een comment toevoegen van wat de cache betekent: `// 1 minute`
Dan zijn alle opties duidelijk. 
Een blokje eronder met "Manually clearing cache" (oid) en een regel `dispatch(invalidateApiDataRequest...` is dan voldoende